helm/charts/determined/values.yaml

```diff
@@ -128,3 +128,6 @@ telemetry:
 
 ## A user friendly name to identify this cluster by.
 # clusterName: Dev
+
+# This is the default password for the admin and determined accounts
+defaultPassword:
```

Comment:
Can we also document this in: https://docs.determined.ai/latest/reference/helm-config.html

Also make sure to add a blank line to all of the files :)